src/config.ts

```diff
@@ -5,4 +5,5 @@ export const config = {
   twitterUrl: 'https://twitter.com/shorebirddev',
   docsUrl: 'https://docs.shorebird.dev',
   contactEmail: 'contact@shorebird.dev',
+  calendly: 'https://calendly.com/eseidel/30m',
```

Comment:
@eseidel lmk if you prefer to use a different link

Reply:
Yeah, I'll get a different link specific for this, sec.

Reply:
https://calendly.com/eseidel/shorebird-demo